### DISCUSSION TITLE
How to reduce the number of data point drawn on line and area of Apex charts?

### DISCUSSION THREAD

**Discussion by swatinawale27** [apexcharts/apexcharts.js#4456] [Q&A]
I have a dataset and we are going to use line/apex chart to plot it. However, dataset can be large and reach 2000+. When all the data is plot on chart, it gets a bit clumsy. Is there any way that ApexCharts can make some data decimation and limit data points to 200 to 500.

**Comment by zee0786** [comment]
did you try grouping the data and club into smaller chunk before passing to the chart. It might solve your problem

**Reply by swatinawale27** [reply]
Yes, i tried. Even after grouping data points are 2000+.

For instance, chart-js offers this kind of data management which is exactly what I want to do with apexchart
https://www.chartjs.org/docs/master/samples/advanced/data-decimation.html
